Doc/library/stdtypes.rst

```diff
@@ -39,31 +39,26 @@ Truth Value Testing
    single: false
 
 Any object can be tested for truth value, for use in an :keyword:`if` or
-:keyword:`while` condition or as operand of the Boolean operations below. The
-following values are considered false:
+:keyword:`while` condition or as operand of the Boolean operations below.
 
-  .. index:: single: None (Built-in object)
-
-* ``None``
-
-  .. index:: single: False (Built-in object)
-
-* ``False``
-
-* zero of any numeric type, for example, ``0``, ``0.0``, ``0j``.
+.. index:: single: true
 
-* any empty sequence, for example, ``''``, ``()``, ``[]``.
+By default, an object is considered true unless its class defines either a
+:meth:`__bool__` method that returns ``False`` or a :meth:`__len__` method that
+returns zero, when called with the object. [1]_  Here are most of the built-in
+objects considered false:
 
-* any empty mapping, for example, ``{}``.
+  .. index::
+     single: None (Built-in object)
+     single: False (Built-in object)
 
-* instances of user-defined classes, if the class defines a :meth:`__bool__` or
-  :meth:`__len__` method, when that method returns the integer zero or
-  :class:`bool` value ``False``. [1]_
+* constants defined to be false: ``None`` and ``False``.
 
-.. index:: single: true
+* numeric zero of any type: ``0``, ``0.0``, ``0j``, ``Decimal(0)``,
+  ``Fraction(0, 1)``
```

Comment:
I agree with changing back to 'zero of any numeric type'.  Peter, change this (I cannot, for some reason) and I will merge.